model/src/w3gridmd.F90

```diff
@@ -2460,33 +2460,25 @@ SUBROUTINE W3GRID()
        B_JGS_NLEVEL = JGS_NLEVEL
        B_JGS_SOURCE_NONLINEAR = JGS_SOURCE_NONLINEAR
 
-       IF ((EXPFSN .eqv. .FALSE.).and.(EXPFSPSI .eqv. .FALSE.)     &
```

Comment:
I think this needs to also be included with https://github.com/NOAA-EMC/WW3/issues/644 

I'll try removing this if statement as well as part of the default updates.

Reply:
hmm, yes we can go ahead like this ...

Reply:
Removing this statement was essential to getting greater consistency when separating out the change in defaults.

Reply:
ok